docs/additional-functionality/rapids-udfs.md

```diff
@@ -134,29 +134,28 @@ implements a Hive simple UDF using
 [native code](../../udf-examples/src/main/cpp/src) to count words in strings
 
 
-## GPU Scheduling For Pandas UDF
+## GPU support for Pandas UDF
 
 ---
 **NOTE**
 
-The _GPU Scheduling for Pandas UDF_ is an experimental feature, and may change at any point it time.
+The _GPU support for Pandas UDF_ is an experimental feature, and may change at any point it time.
```

Comment:
Italics are out of place here, "GPU support for Pandas UDF" is not an official name for anything.
```suggestion
The GPU support for Pandas UDF is an experimental feature, and may change at any point it time.
```

Reply:
Updated.